### ISSUE TITLE
Compilation error on MinGW (gcc 4.9.2) and gcc 4.8.2

### ISSUE THREAD

**Issue by Squareys** [mosra/magnum#100]
Hello @mosra, 

with https://github.com/mosra/magnum/commit/387f3b78d6f13bc80c980f517c38e7b4f268ba03 I now get a compile time error:

```
/home/squareys/repos/magnum/src/Magnum/Context.cpp: In constructor ‘Magnum::Context::Context(Magnum::Context&&)’:
/home/squareys/repos/magnum/src/Magnum/Context.cpp:554:55: error: cannot convert ‘std::remove_reference<std::array<Magnum::Version, 160ul>&>::type {aka std::array<Magnum::Version, 160ul>}’ to ‘Magnum::Version’ in initialization
     _detectedDrivers{std::move(other._detectedDrivers)}
```

On both `MinGW (i686-posix-dwarf using gcc 4.9.2)` and `gcc 4.8.2 (Ubuntu 4.8.2-19ubuntu1)`.

Greetings, 
Squareys

**Comment by mosra** [comment]
Oh hello, another GCC < 5.1 bug :)

changing the `{}` to `()` should fix that:
```cpp
_detectedDrivers(std::move(other._detectedDrivers))
```

I'll commit the fix later today. Thanks a lot for the report!

**Comment by mosra** [comment]
Fixed in 87e964ce69113142de3a470876833b4991c0a7e7. Oh and congrats on creating issue #100! :)

**Comment by Squareys** [comment]
Thanks! Congrats on resolving issue #100! ;)
